poetry.lock

```diff
@@ -650,15 +658,15 @@ pyparsing = ">=2.0.2"
 
 [[package]]
 name = "pandas"
-version = "0.25.3"
+version = "1.1.5"
```

Comment:
`pandas >= 1.0.0` only supports `Python ^=3.6.1`. I think there are very few users who use `Python ==3.6.0` exactly, so I think it's not relevant for many users. The older pandas versions did not install nicely on my system with 3.9 (or would have required additional build libraries, as pandas 0.25.3 [does not have wheels for 3.9](https://pypi.org/project/pandas/0.25.3/#files).